src/Webapi/Webapi__Dom/Webapi__Dom__HtmlDocument.re

```diff
@@ -3,11 +3,13 @@ module Impl = (T: {type t;}) => {
 
   [@bs.get] [@bs.return nullable] external activeElement : t_htmlDocument => option(Dom.element) = "";
   [@bs.get] [@bs.return nullable] external body : t_htmlDocument => option(Dom.element) = ""; /* returns option HTMLBodyElement */
+  [@bs.get] external characterSet : t_htmlDocument => string = "";
   [@bs.set] external setBody : (t_htmlDocument, Dom.element) => unit = "body"; /* accepth HTMLBodyElement */
   [@bs.get] external cookie : t_htmlDocument => string = "";
   [@bs.set] external setCookie : (t_htmlDocument, string) => unit = "cookie";
   [@bs.get] [@bs.return nullable] external defaultView : t_htmlDocument => option(Dom.window) = "";
   [@bs.get] external designMode : t_htmlDocument => string /* designMode enum */ = "";
+  [@bs.get] external contentType: T.t => string = "";
```

Comment:
This is marked as experimental on MDN and seems to be unavailable on IE11 and Opera Mini. I'm hesitant to include APIs that aren't fully standardized, although this seems pretty established. It should at least have a comment noting that it's experimental here as well. It should also be in `Document`, not `HtmlDocument`.